packages/RichText-Core.package/RichTextEditor.class/instance/initialize.st

```diff
@@ -1,14 +1,11 @@
 initialization
 initialize
 
-	| wrapper |
-
 	super initialize.
 	self
 		extent: 500@200;
 		
 		flag: #refactor;
 		"adding the addComponent should be handled by the Layout component itself!"
-		coreObject: (RTEVerticalLayoutComponent new addComponent: RTEAddComponent).
-	wrapper := self initializeWrapper.
-	self content: wrapper
\ No newline at end of file
+		coreObject: (RTEVerticalLayoutComponent new addComponent: RTEAddComponent);
```

Comment:
and here. I think you removed almost any new feature :D

Reply:
I think you should undo the last commit and merge again. There is so much wrong here